pkg/cmd/registry/rule/rulecmdutil/constants.go

```diff
@@ -0,0 +1,17 @@
+package rulecmdutil
+
+const (
+	ValidityRule      = "validity"
+	CompatibilityRule = "compatibility"
+)
+
+const (
+	ConfigFULL                = "full"
```

Comment:
Should  we have those defined in the OpenAPI/SDK?

Reply:
These  are just for convenience of cli user for not having to toggle between caps. This won't be something we would want in sdk.